### ISSUE TITLE
ghcid crashes/exits if .cabal is edited to an invalid state

### ISSUE THREAD

**Issue by I3ck** [ndmitchell/ghcid#198]
Displaying something similar to   
".cabal invalid"  
or  
"ghci can't be run"  
would be better instead. Fixing the .cabal could then result in ghcid running properly again.

**Comment by ndmitchell** [comment]
Makes sense. More concretely, if you don't get into a `ghci` prompt and have `--restart` flags it should tell you what you might need to do to restart things.
